site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -1,43 +1,72 @@
 ==== Request Body
 
+// SAMLv2 idp initiated has a subset of the attributes of the normal SAML IdP so just omitting the extra attributes for that idp.
+// do the check on every attribute for clarity
+
 [.api]
 [field]#identityProvider.applicationConfiguration# [type]#[Map<UUID, Object>]# [optional]#Optional#::
 The configuration for each Application that the identity provider is enabled for.
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.applicationConfiguration``[applicationId]``.buttonImageURL# [type]#[String]# [optional]#Optional#::
 This is an optional Application specific override for the top level [field]#buttonImageURL#.
+endif::[]
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.applicationConfiguration``[applicationId]``.buttonText# [type]#[String]# [optional]#Optional#::
 This is an optional Application specific override for the top level [field]#buttonText#.
+endif::[]
 
 [field]#identityProvider.applicationConfiguration``[applicationId]``.createRegistration# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `true`#::
 Determines if a UserRegistration is created for the User automatically or not. If a user doesn't exist in FusionAuth and logs in through an identity provider, this boolean controls whether or not FusionAuth creates a registration for the User in the Application they are logging into.
 
 [field]#identityProvider.applicationConfiguration``[applicationId]``.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
 Determines if this identity provider is enabled for the Application specified by the `applicationId` key.
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.buttonImageURL# [type]#[String]# [optional]#Optional#::
 The top-level button image (URL) to use on the FusionAuth login page for this Identity Provider.
+endif::[]
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.buttonText# [type]#[String]# [required]#Required#::
 The top-level button text to use on the FusionAuth login page for this Identity Provider.
+endif::[]
 
+ifndef::samlv2_idp_initiated[]
 [field]#identityProvider.debug# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.7.3#::
 Determines if debug is enabled for this provider. When enabled, each time this provider is invoked to reconcile a login an Event Log will be created.
 +
 This is useful for debugging purposes, but is not intended to be left on during production. It should be enabled during integration or if you are experiencing an issue with this identity provider.
+endif::[]
+ifdef::samlv2_idp_initiated[]
+[field]#identityProvider.debug# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
+Determines if debug is enabled for this provider. When enabled, each time this provider is invoked to reconcile a login an Event Log will be created.
```

Comment:
consider
Determines if debug is enabled for this provider. When enabled, an Event Log is created each time this provider is invoked to reconcile a login.